avocado/utils/nvme.py

```diff
@@ -90,3 +93,220 @@ def get_controller_id(controll_name):
         if "cntlid" in line:
             return line.split(":")[-1].strip()
     return ""
+
+
+def get_current_ns_ids(controll_name):
+    """
+    Returns the list of namespaces in the nvme controller
+
+    :param controller_name: Name of the nvme controller like nvme0, nvme1
```

Comment:
is the input can be comma separated multiple controllers ?

Reply:
It is not multiple controller as an input. I have given there as an example with multiple controller name.
I will change to single controller name if it is confusing.